nameko_amqp_retry/backoff.py

```diff
@@ -94,9 +113,9 @@ def make_queue(self, expiration):
 
     def republish(self, backoff_exc, message, target_queue):
 
-        expiration = backoff_exc.get_next_expiration(
-            message, self.exchange.name
-        )
+        backoff_exc.calculate_next_expiration(message, self.exchange.name)
+        expiration = backoff_exc.next_expiration
```

Comment:
If it does both, sets next and returns (comment above), could return straight away
```
expiration = backoff_exc.next_expiration(message, self.exchange.name)
```